packages/server/test/speckle-helpers/streamHelper.ts

```diff
@@ -132,10 +132,10 @@ export async function createTestStreams(
 }
 
 /**
- * Create basic stream for testing and update streamObj to have a real ID
+ * Create basic stream for testing and update streamObj in-place, via reference, to have a real ID
  */
-export async function createTestStream(
-  streamObj: BasicTestStream,
+export async function createTestStream( //FIXME this function has side-effects, and amends the object in-place
```

Comment:
@iainsproat The side-effects are definitely intentional

Reply:
well, this aspect of the function is a part of its core responsibilities, its pretty important to have it cause manually having to import id-generating functions, having to remember the correct lengths & formats of ids is a PITA that this aspect of the helper resolves. in fact, this was one of the main reasons for me even creating these. it would be a serious impact to DX if we were to remove it.

its not immediately obvious to me how these helpers would cause issues in parallel tests, but if any pop up we can deal with them then